omod/src/main/webapp/resources/scripts/conditionlist/controllers/editcondition.controller.js

```diff
@@ -92,7 +95,12 @@ function ConditionController($scope, RestfulService, ConditionModel, ConceptMode
         var groups = document.getElementsByClassName("group");
         if($scope.condition.status == INACTIVE_STATUS){
             groups[2].style.visibility = "visible";
-            inputs[3].value = CommonFunctions.formatDate($scope.condition.endDate); 
+
+            if($scope.condition.endDate==null){
+                inputs[3].value = CommonFunctions.formatDate(new Date()); 
+                } else {
```

Comment:
```suggestion
            } else {
```

`} else {` and the closing brace should all be in line with the opening line (`if...`)